src/main/java/edu/umich/its/cpm/MigrationController.java

```diff
@@ -1939,9 +1988,12 @@ private void handleBulkResourceBoxMigration(String sessionId,
 						+ siteId + " error message="
 						+ status.get("errorMessage"));
 			} else if (status.containsKey("migrationId")) {
+				
+				String migrationId = status.get("migrationId");
```

Comment:
Why repeat this code rather than use a method? (1899  and 1991).
To my surprise the logger has a simplified formatter which makes writing messages easier.

This statement:
log.info(this + " batch upload call for site id="+ siteId + " migration started id="+ migrationId);

Can become this:
log.info("{} batch upload call for site id={}  migration started id={}", this,siteId,migrationId);

The {} can be used to insert information instead of using string construction or String.format.  It doesn't work for fancy formatting, but that can often be avoided in a log statement.